libraries/AC_PID/AC_PID.cpp

```diff
@@ -161,6 +161,10 @@ float AC_PID::update_all(float target, float measurement, float dt, bool limit)
     P_out *= _pid_info.Dmod;
     D_out *= _pid_info.Dmod;
 
+    // boost output if required
+    P_out *= boost;
+    D_out *= boost;
```

Comment:
Applying the boost here bypasses the slew limiter, I think it should be on line 156 so the slew limiter still works on the boosted output.

Reply:
It is based on the last loop output so the order doesn't really matter.

As we are going to do some work on the slew limiter I would not make this a blocker. I would be tempted to move the slew limiter calculation to the end and use the result in the next loop.

Reply:
No it doesn't